### DISCUSSION TITLE
Link to app in App Store?

### DISCUSSION THREAD

**Discussion by SuzGupta** [mikaelacaron/Basic-Car-Maintenance#121] [General & Questions] [answered]
Is there an App Store version of this right now? I'm curious what the listing looks like, as this is something I can definitely help with.

**Comment by mikaelacaron** [comment, marked as answer]
Not at the moment! I had planned on uploading it in the first week, but there's actually a few critical things that need to be fixed first.
I'd love for you to make an issue with what the App Store listing should look like, and keywords I could use! ❤️ 

haha I'll remove this from the README cause it's looking like I won't publish this until after October (Hacktoberfest is over)

**Reply by SuzGupta** [reply]
I'll add this as an issue right now and then expand on it. Happy to create this in App Store Connect if you want to add me to whichever account you plan to use. I wonder if we can get Appfigures to give us a free account for keyword research since this is open source. Working through how to set up Name, subtitle and keywords and just generally make a great listing could make a fun and useful video.

**Reply by mikaelacaron** [reply]
That's a good idea! I'll reach out to Appfigures! I'll DM you about the App Store Connect stuff (but I'm not going to be doing this, at the moment, because I don't think this needs to happen immediately).

Adding a page to the DocC documentation for the description, keywords etc, I think is fine for now
